tools/msconvert/msconvert_macros.xml

```diff
@@ -551,7 +551,7 @@
   <xml name="msconvertOutput">
     <outputs>
       <data format="mzml" name="output" label="${($input.name[:-4] if $input.name.endswith('.tar') else $input.name).rsplit('.',1)[0]}.${output_type}" >
-        <filter>general_options['multi_run_output']['do_multi_run_output'] == False</filter>
+        <filter>general_options['multi_run_output']['do_multi_run_output'] == "false"</filter>
```

Comment:
This looks wrong if its a boolean.

Reply:
This was changed to a select option in version 3.0.20287.3 (https://github.com/galaxyproteomics/tools-galaxyp/pull/748).